server/planning/__init__.py

```diff
@@ -321,7 +326,9 @@ def init_scheduled_exports_task(app):
 
 @celery.task(soft_time_limit=600)
 def flag_expired():
-    FlagExpiredItems().run()
+    import asyncio
```

Comment:
You should be able to simply prefix the function definition with `async ` as we added async support to Celery tasks, and used in [Newshub](https://github.com/superdesk/newsroom-core/blob/async/newsroom/push/tasks.py#L31).

That way we don't have to worry about running `asyncio.run` or our own event loop. If this does't work here for some reason, then that's a different issue that needs to be looked at for the Planning module